README.md

```diff
@@ -286,6 +312,11 @@ Set GPIO activeLow setting.
 Reverse the effect of exporting the GPIO to userspace. A Gpio object should not
 be used after invoking its unexport method.
 
+##### static accessible
+True if your system supports GPIO pins. It checks if there are no errors in opening your GPIO export path.
```

Comment:
The onoff documentation shouldn't use the words "you", your", "I", etc... The text here should be reformulated to avoid using these words.